### PR TITLE
Prepare the FAQ for archiving

### PR DESCRIPTION
**Pull request by Gudahtt**
All files have been removed apart from the README, and the README has been emptied of information aside from the archive notice and the link to our current documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,4 @@
-**This FAQ may be outdated. Find our most up-to-date documentation [here](https://metamask.github.io/metamask-docs/)**
+# This FAQ has been archived.
 
-# MetaMask FAQ
+## Find our most up-to-date documentation [here](https://metamask.github.io/metamask-docs/)
```
Comment (rekmarks) [inline README.md:3]: We should probably link to the live site:
```suggestion
## Find our most up-to-date documentation [here](https://docs.metamask.io)
```

**Review by rekmarks — CHANGES_REQUESTED** [review]
Nice initiative!

File: README.md (hunk 1 above)
Comment (Gudahtt) [reply README.md:3]: Ah right, good catch!

**Review by rekmarks — APPROVED** [review]
LGTM!

**Pull request merged**